shuler_lab_to_nwb/fiberphotometry_datainterface/tools.py

```diff
@@ -0,0 +1,117 @@
+from typing import Optional
+import pandas as pd
+from hdmf.backends.hdf5 import H5DataIO
+from hdmf.common import DynamicTableRegion
+from ndx_photometry import FibersTable, FiberPhotometry, ExcitationSourcesTable, PhotodetectorsTable, FluorophoresTable
+from pynwb import NWBFile
+from pynwb.ophys import RoiResponseSeries
+
+
+def read_fp_file(file_path: str) -> pd.DataFrame:
```

Comment:
For readability, recommend enhancing method name to 

```suggestion
def read_fibre_photometry_csv_file(file_path: str) -> pd.DataFrame:
```

or something similar